config/locales/devise.en.yml

```diff
@@ -14,7 +14,6 @@ en:
       last_attempt: "You have one more attempt before your account is locked."
       not_found_in_database: "Invalid %{authentication_keys} or password."
       timeout: "Your session expired. Sign in to continue."
-      unauthenticated: "Sign in to continue"
```

Comment:
it's not as simple as that. The value needs to be an empty string if you want to display nothing. If the key is missing, it will default to devise's locale: https://github.com/plataformatec/devise/blob/43068ac23907cbf8f74dfedd60fe4e9f14560006/config/locales/en.yml#L17